src/pages/Checkout/CheckoutPage.vue

```diff
@@ -73,7 +72,6 @@
 							:promo-fund="derivedPromoFund"
 							@refreshtotals="refreshTotals"
 							@updating-totals="setUpdatingTotals"
-							:show-matched-loan-kiva-credit="showMatchedLoanKivaCredit && requireDepositsMatchedLoans"
```

Comment:
Looks like `showMatchedLoanKivaCredit` computed can be removed

Reply:
Done